spec/support/helpers/factory_girl.rb

```diff
@@ -1,7 +1,7 @@
-module TimeHelpers
-  def self.quartile_minute(time)
+module FactoryGirl::Syntax::Methods
```

Comment:
yes! found this method via https://github.com/thoughtbot/factory_girl/issues/564#issuecomment-157669866